### DISCUSSION TITLE
How are features decided if they are enterprise or open source or somewhere in between?

### DISCUSSION THREAD

**Discussion by deefdragon** [Flagsmith/flagsmith#3455] [General] [answered]
I'm curious how features are decided to be restricted to the different variants vs made open source etc.

The reason for my curiosity: I added a second user to flagsmith so they could update a flag while I was out if things started breaking. As a security conscious individual, I had set all of their permissions so they only had access to the one environment. I also decided to update my flagsmith helm chart while I was thinking about it. After upgrading, I come to find that I can no longer adjust the users permissions, and am met with `Role-based access is not available on our Free Plan. Please visit [our Pricing Page](https://flagsmith.com/pricing/) for more information on our licensing options` **_(this is NOT a discussion about this particular situation, I'm just including for context)_**

Initially this had me paranoid at more features being made enterprise only, but I'm assuming I was not supposed to be able to do what I did to begin with, and honestly I'm now more curious at how features are decided to be made enterprise vs open source etc.

**Comment by dabeeeenster** [comment, marked as answer]
Hi There, good question!

A couple of things to bring up first here:

1. RBAC wasnt in the open source version at any point - its possible either you thought it was because we werent clear about this in older versions UI, or there was a misconfiguration. Point being that it wasnt a feature that was open and moved to closed. 
2. We haven't ever moved a feature from open to closed source, and we don't ever plan on doing that.

We do have a core philosophy around this that is public here: https://docs.google.com/presentation/d/1IDP56WALkuPIUg467blTgdbDNT6V_aTObKTonfhxR8o/edit?usp=sharing

Would love any feedback on this - it's an important topic!
